triton/plan.sh

```diff
@@ -0,0 +1,38 @@
+pkg_name=triton
+pkg_origin=core
+pkg_maintainer="The Habitat Maintainers <humans@habitat.sh>"
+pkg_license=('MPL-2.0')
+pkg_deps=(core/node)
+pkg_build_deps=(core/jq-static core/sed)
+pkg_bin_dirs=(bin)
+pkg_description="Joyent Triton CLI and client"
+pkg_upstream_url="https://www.joyent.com/triton"
+
+pkg_version () {
+  npm view triton --json | jq --raw-output .version
```

Comment:
I think the latest version is fine for applications that you own; but this kind of scares me if we don't ever validate the sha.  What if their binary is compromised somehow and then people end getting infected versions.  I'm just paranoid.